app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+def copy_file(command: str) -> None:
+    list_from_command = command.split(" ")
+    command_name, file_name, copy_file_name = list_from_command
```

Comment:
Consider using more descriptive variable names for better readability. For example, `command_name` could be `operation`, `file_name` could be `source_file`, and `copy_file_name` could be `destination_file` as suggested in the checklist .